### DISCUSSION TITLE
chinese files

### DISCUSSION THREAD

**Discussion by lucky-zhao** [minio/minio#13863] [Q&A] [answered]
version : minio/minio:latest

The picture with Chinese name failed to upload and cannot be previewed
![image](https://user-images.githubusercontent.com/45654758/145173945-f96d5d47-6f60-45c0-a781-2d132ff0299c.png)
I looked at the log and output: The specified key does not exist.

English name of the picture, everything is normal, you can download, you can preview
![image](https://user-images.githubusercontent.com/45654758/145173845-199d69d6-c0e8-453a-b138-e51efaeacd51.png)

**Comment by lucky-zhao** [comment]
chinese name can not download and preview

**Comment by harshavardhana** [comment, marked as answer]
This is actually fixed in the console, wait for next release https://github.com/minio/console/pull/1296

```
commit 3ce0b3d6338a0fed430d1b7ca2779d969d177b50
Author: Lenin Alevski <alevsk.8772@gmail.com>
Date:   Mon Dec 6 13:49:13 2021 -0800

    Fixed share/download object regression (#1296)
    
    * Fixed share/download object regression
    * Adding tests for computeObjectURLWithoutEncode function
    
    Signed-off-by: Lenin Alevski <alevsk.8772@gmail.com>
```

**Comment by lucky-zhao** [comment]
I looked at it and found that the uploaded pictures with Chinese names turned into a folder. Why did I turn my uploaded pictures into a folder name。
![image](https://user-images.githubusercontent.com/45654758/145176798-b2036c8f-c792-498c-88b4-9b17d5bc2f5a.png)

**Reply by harshavardhana** [reply]
> I looked at it and found that the uploaded pictures with Chinese names turned into a folder. Why did I turn my uploaded pictures into a folder name。

That is called erasure-coded data format https://github.com/minio/minio/tree/master/docs/erasure

**Comment by lucky-zhao** [comment]
Which version should I use now. I need to upload a file with Chinese name

**Reply by harshavardhana** [reply]
> Which version should I use now. I need to upload a file with Chinese name

@lucky-zhao wait for new MinIO release or you build it yourself from source.

**Reply by lucky-zhao** [reply]
> 



> > Which version should I use now. I need to upload a file with Chinese name
> 
> @lucky-zhao wait for new MinIO release or you build it yourself from source.

ok,thinks
